msos/WaitChains.cs

```diff
@@ -108,7 +108,11 @@ private void DisplayChainForThread(UnifiedThread unifiedThread, int depth, HashS
         {
             foreach (var blockingObject in unifiedThread.BlockingObjects)
             {
-                _context.Write("{0}| {1} ", new string(' ', (depth + 1) * 2), blockingObject.Reason);
+                string reason = blockingObject.Type == UnifiedBlockingType.ThreadSleep
+                                ? $"ThreadSleep: {blockingObject.SleepWait.Value}"
+                                : blockingObject.Reason.ToString();
```

Comment:
I added ReasonDescription property in UnifiedBlockingObject class, previously onlyReason property (which is an UnifiedBlockingReason enum) was printed.